ynput/ayon-core
# Unreal: add support of PySide6 in Unreal 5.4

## Changelog Description
Unreal 5.4 comes with Python 3.11 and because of that it needs PySide6. This is fixing installation of Qt bindings to allow that.

## Testing notes:
1. Install Unreal 5.4 and configure it in AYON
2. Create new project (run it on empty task) -> it should start up normally
